meson.build

```diff
@@ -939,7 +939,7 @@ if features['sdl2-video']
     sources += files('video/out/vo_sdl.c')
 endif
 
-shaderc = dependency('shaderc', required: get_option('shaderc'))
+shaderc = dependency('shaderc', required: get_option('shaderc').require(features['win32-desktop']))
```

Comment:
why not have it depend on d3d11?

Reply:
It's kind of circular. d3d11 depends on shaderc currently.

Reply:
I suppose there's no way around it.
By the way, the same applies to `spirv-cross`.

Reply:
Included spirv-cross in the first commit as well.